tests/providers/http/operators/test_http_system.py

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+
+import pytest
+
+from tests.test_utils import AIRFLOW_MAIN_FOLDER
+from tests.test_utils.system_tests_class import SystemTest
+
+HTTP_DAG_FOLDER = os.path.join(
+    AIRFLOW_MAIN_FOLDER, "airflow", "providers", "http", "example_dags"
+)
+
+
+@pytest.mark.backend("mysql", "postgres")
+@pytest.mark.system("http")
```

Comment:
I'm wondering if we should use more generic system type like "core" or something. WDYT? @potiuk ?

Reply:
It is not the core operator. It is in the provider directory. This allows it to be released independently from Airflow.  It is common and mature, but its use and usage are still developing. It is flexible enough to be extended with new authentication mechanisms and more. Last week we had a valuable and non-doc contribution to this operator.
https://github.com/apache/airflow/pull/8429

Here is list of all core operators.
```
airflow.operators.bash.BashOperator
airflow.operators.branch_operator.BaseBranchOperator
airflow.operators.check_operator.CheckOperator
airflow.operators.check_operator.IntervalCheckOperator
airflow.operators.check_operator.ThresholdCheckOperator
airflow.operators.check_operator.ValueCheckOperator
airflow.operators.dagrun_operator.TriggerDagRunOperator
airflow.operators.dummy_operator.DummyOperator
airflow.operators.generic_transfer.GenericTransfer
airflow.operators.latest_only_operator.LatestOnlyOperator
airflow.operators.presto_check_operator.PrestoCheckOperator
airflow.operators.presto_check_operator.PrestoIntervalCheckOperator
airflow.operators.presto_check_operator.PrestoValueCheckOperator
airflow.operators.python.BranchPythonOperator
airflow.operators.python.PythonOperator
airflow.operators.python.PythonVirtualenvOperator
airflow.operators.python.ShortCircuitOperator
airflow.operators.subdag_operator.SubDagOperator
```

Reply:
Agree with Kamil 👍